src/components/StyleGuide/StyleGuide.scss

```diff
@@ -12,12 +12,36 @@
 
 .sectionText {
   @include type-h1;
+
+  margin-bottom: 16px;
 }
 
 .subSectionText {
   @include type-h3;
 }
 
+.iconsWrapper {
+  margin-bottom: 64px;
+}
+
+.iconWrapper {
+  @include shadow;
+
+  display: inline-block;
+  padding: 16px 0;
+  margin: 0 16px 16px 0;
+  width: 200px;
+  text-align: center;
+  border-radius: 4px;
+  color: $color-primary--500;
```

Comment:
Colour the icons just so it's clear that the 'P' in the circle icon is hard-coded grey.